formulas/slug.txt

```diff
@@ -1,191 +1,30 @@
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-SUBSTITUTE(
-TRIM(LOWER(TODO_REPLACE_WITH_YOUR_COLUMN_NAME)),
-"à", "a"),
-"á", "a"),
-"â", "a"),
-"ä", "a"),
-"æ", "a"),
-"ã", "a"),
-"å", "a"),
-"ā", "a"),
-"æ", "ae"),
-"ç", "c"),
-"ć", "c"),
-"č", "c"),
-"è", "e"),
-"é", "e"),
-"ê", "e"),
-"ë", "e"),
-"ē", "e"),
-"ė", "e"),
-"ę", "e"),
-"î", "i"),
-"ï", "i"),
-"í", "i"),
-"ī", "i"),
-"į", "i"),
-"ì", "i"),
-"ł", "l"),
-"ñ", "n"),
-"ń", "n"),
-"ô", "o"),
-"ö", "o"),
-"ò", "o"),
-"ó", "o"),
-"œ", "o"),
-"ø", "o"),
-"ō", "o"),
-"õ", "o"),
-"ß", "s"),
-"ś", "s"),
-"š", "s"),
-"û", "u"),
-"ü", "u"),
-"ù", "u"),
-"ú", "u"),
-"ū", "u"),
-"ÿ", "y"),
-"ž", "z"),
-"ź", "z"),
-"ż", "z"),
-" ", "-"),
-"!", ""),
-".", ""),
-"/", ""),
-"”", ""),
-"'", ""),
-"#", ""),
-"$", ""),
-"%", ""),
-"&", ""),
-"’", ""),
-"(", ""),
-")", ""),
-"*", ""),
-"+", ""),
-",", ""),
-":", ""),
-";", ""),
-"<", ""),
-"=", ""),
-">", ""),
-"?", ""),
-"@", ""),
-"[", ""),
-"\\", ""),
-"]", ""),
-"^", ""),
-"_", ""),
-"`", ""),
-"{", ""),
-"|", ""),
-"}", ""),
-"~", ""),
-"«", ""),
-"»", ""),
-"€", ""),
-"„", ""),
-"“", ""),
-"\"", ""),
-"---------", "-"),
-"--------", "-"),
-"-------", "-"),
-"------", "-"),
-"-----", "-"),
-"----", "-"),
-"---", "-"),
-"--", "-")
+REGEX_REPLACE(
+REGEX_REPLACE(
+REGEX_REPLACE(
+REGEX_REPLACE(
+REGEX_REPLACE(
+REGEX_REPLACE(
+REGEX_REPLACE(
+REGEX_REPLACE(
+REGEX_REPLACE(
+REGEX_REPLACE(
+REGEX_REPLACE(
+REGEX_REPLACE(
+REGEX_REPLACE(
+REGEX_REPLACE(
+    LOWER(TRIM(TODO_REPLACE_WITH_YOUR_COLUMN_NAME)
+), '\s+', '-' 
+), '[àáâäæãåā]+', 'a'
+), '[æ]+', 'ae'
+), '[çćč]+', 'c'
+), '[èéêëēėę]+', 'e'
+), '[îïíīįì]+', 'i'
+), '[ñń]+', 'n'
+), '[öòóœøōõ]+', 'o'
+), '[ßśš]+', 's'
+), '[ùûüúū]+', 'u'
+), '[ÿ]+', 'y'
+), '[žźż]+', 'y'
+), '[!./”\'#$%&’()*+,:;<=>?@[\\]^_`{|}~«»€„“]+', ''
+), '[\--]+', '-'
```

Comment:
`\--` does that mean it will replace an unlimited amount of occurrences of `-` if there are more than 2?